src/realm/sync/noinst/pending_bootstrap_store.cpp

```diff
@@ -198,7 +202,13 @@ bool PendingBootstrapStore::has_pending()
 
 void PendingBootstrapStore::clear()
 {
-    auto tr = m_db->start_write();
+    auto tr = m_db->start_read();
+    auto bootstrap_table = tr->get_table(m_table);
+    // Just make sure the state is reset if the bootstrap table is empty
```

Comment:
not sure how this helps (or what the comment means)

Reply:
Decided to revert this change.